packages/php73.rb

```diff
@@ -60,45 +61,44 @@ def self.patch
   end
 
   def self.build
-    ENV['TMPDIR'] = "#{CREW_PREFIX}/tmp"
-    ENV['CFLAGS'] = ' -liconv'
-    system './configure',
-           "--prefix=#{CREW_PREFIX}",
-           "--docdir=#{CREW_PREFIX}/doc",
-           "--infodir=#{CREW_PREFIX}/info",
-           "--libdir=#{CREW_LIB_PREFIX}",
-           "--localstatedir=#{CREW_PREFIX}/tmp",
-           "--mandir=#{CREW_PREFIX}/share/man",
-           "--sbindir=#{CREW_PREFIX}/bin",
-           "--with-config-file-path=#{CREW_PREFIX}/etc",
-           "--with-libdir=#{ARCH_LIB}",
-           "--with-kerberos=#{CREW_LIB_PREFIX}",
-           '--enable-exif',
-           '--enable-fpm',
-           '--enable-ftp',
-           '--enable-mbstring',
-           '--enable-opcache',
-           '--enable-pcntl',
-           '--enable-shared',
-           '--enable-shmop',
-           '--enable-sockets',
-           '--enable-zip',
-           '--with-bz2',
-           '--with-curl',
-           '--with-gd',
-           '--with-gettext',
-           '--with-gmp',
-           '--with-libzip',
-           '--with-mysqli',
-           '--with-openssl',
-           '--with-pcre-regex',
-           '--with-pdo-mysql',
-           '--with-pear',
-           '--with-readline',
-           '--with-tidy',
-           '--with-unixODBC',
-           '--with-xsl',
-           '--with-zlib'
+    system "env TMPDIR='#{CREW_PREFIX}/tmp' CFLAGS='-pipe -ltinfo' \
+      ./configure \
+       --prefix=#{CREW_PREFIX} \
+       --docdir=#{CREW_PREFIX}/doc \
+       --infodir=#{CREW_PREFIX}/info \
+       --libdir=#{CREW_LIB_PREFIX} \
+       --localstatedir=#{CREW_PREFIX}/tmp \
+       --mandir=#{CREW_PREFIX}/share/man \
```

Comment:
This could be `--mandir=#{CREW_MAN_PREFIX}`.

Reply:
Can you confirm this?  Just googling "php mandir share/man" shows a lot of usage where it is prefix/share/man...

Reply:
OH, never mind.